modules/k6/examples_test.go

```diff
@@ -0,0 +1,73 @@
+package k6_test
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/k6"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+func ExampleRunContainer() {
+	// runK6Container {
+	ctx := context.Background()
+
+	gcr := testcontainers.GenericContainerRequest{
+		ProviderType: testcontainers.ProviderDocker,
+		ContainerRequest: testcontainers.ContainerRequest{
+			Image: "kennethreitz/httpbin",
```

Comment:
Let's use a fixed tag for this image if possible.

Reply:
As far as I know, there are no tags for this image, which is widely used for testing.

https://hub.docker.com/r/kennethreitz/httpbin/tags

Reply:
Cool than. Let's be specific about using `latest` then 🙏